README.md

```diff
@@ -108,14 +154,20 @@ npm install
 
 ### Working with React
 
-#### Build the project bundle
+The frontend code could be found in `webapp-admin` directory.
+
+#### Build the project bundle for the local development
 ```bash
 # Select the version to use
+cd webapp-admin/
 nvm ls
 nvm use lts/*
 npm run build
 ```
 
+#### Build the project bundle for the production
+Production build for the frontend part (javascript and css) is semi-automated and done with [github actions](https://github.com/features/actions). You can find the configuration for this process in `.github/workflows/nodejs.yml`. The build process starts automatically after the PR is approved and pushed to `master`. You can see the build process in [actions](https://github.com/CanonicalLtd/serial-vault/actions) tab of this project. After the successful build the automation bot will create a PR with the generated build artifact (minified javascript code) in the `static/` directory of this project. This PR should be approved manually by the developer.
```

Comment:
Maybe we don't need the last sentence? ("This PR should be approved manually by the developer.")

Or change it to: "These PRs can be merged manually."?